hack/verify-staticcheck.sh

```diff
@@ -13,7 +13,18 @@ source "hack/util.sh"
 
 util::install_tools ${GOLANGCI_LINT_PKG} ${GOLANGCI_LINT_VER}
 
-if golangci-lint run; then
+
+function lint() {
+  local ret=0
+
+  golangci-lint run || ret=1
+
+  golangci-lint run --build-tags=tools ./hack || ret=1
```

Comment:
`--build-tags strings             Build tags`
Can you explain what the `--build-tags` flags used for?  I'd prefer to add a comment on it.

Reply:
sure.